<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWindow</class>
 <widget class="QWidget" name="ResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,2">
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblSrc">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Sources</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QTableWidget" name="tblTime">
       <column>
        <property name="text">
         <string>staying in system</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>waiting</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>service</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTableWidget" name="tblRejection">
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>total requests</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>rejected</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>probability</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblCnmrs">
       <property name="text">
        <string>Consumers</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line_3">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="5">
      <widget class="QTableWidget" name="tblConsumers">
       <column>
        <property name="text">
         <string>working time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>work coefficient</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QTableWidget" name="tblDispersion">
       <column>
        <property name="text">
         <string>waiting</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>service</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Rejection</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblTime">
       <property name="text">
        <string>Average time</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblDispersion">
       <property name="text">
        <string>Dispersion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblRealisationTime">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Realisation time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRealisationTimeField">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
